### ISSUE TITLE
Nightly 0.12.0, Linux: Adding song folder freezes the game

### ISSUE THREAD

**Issue by JapanYoshi** [YARC-Official/YARG#644]
I want to add a song folder to v0.12.0 Nightly on Linux (Linux Mint distro). I click "New folder", then click "Browse". I navigate to my song folder and press "Open", but then the explorer pop-up window and game both freeze, and all I can do is force quit.

**Comment by EliteAsian123** [comment]
Can you send the log files?

**Comment by JapanYoshi** [comment]
[Player.log](https://github.com/YARC-Official/YARG/files/13815681/Player.log)
Do you mean this?

**Comment by EliteAsian123** [comment]
Yes, that's it. The space should be fine, but I'm not sure about the non-ascii characters. It seems like GTK is crashing rather than YARG, so it's probably something along those lines, I assume.

**Comment by JapanYoshi** [comment]
Tried again today, not resolved on stable v0.12.1
